reactfire/firebaseApp/sdk.tsx

```diff
@@ -119,100 +123,97 @@ function useSDK(sdk: SDK, firebaseApp?: firebase.app.App) {
 }
 
 export function preloadAuth(
-  firebaseApp: firebase.app.App,
-  settingsCallback?: (auth: () => Auth) => void
```

Comment:
Can I infer from tyepscript?

Reply:
Also should revert regardless, since I can't guarantee side-effects haven't been stripped.